tools/SendBlobs/Program.cs

```diff
@@ -0,0 +1,394 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Microsoft.Extensions.CommandLineUtils;
+using Nethermind.Cli;
+using Nethermind.Cli.Console;
+using Nethermind.Consensus;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Crypto;
+using Nethermind.Evm;
+using Nethermind.Facade.Proxy.Models;
+using Nethermind.Int256;
+using Nethermind.Logging;
+using Nethermind.Serialization.Json;
+using Nethermind.Serialization.Rlp;
+using Org.BouncyCastle.Utilities.Encoders;
+using SendBlobs;
+
+// send-blobs <url-without-auth> <transactions-send-formula 10x1,4x2,3x6> <secret-key> <receiver-address>
+// send-blobs http://localhost:8545 5 0x0000000000000000000000000000000000000000000000000000000000000000 0x000000000000000000000000000000000000f1c1 100 100
+// 1 = 0 blobs
+// 2 = 1st blob is of wrong size
+// 3 = 7 blobs
+// 4 = 1st blob's wrong proof
+// 5 = 1st blob's wrong commitment
+// 6 = 1st blob with a modulo correct, but > modulo value
+// 7 = max fee per blob gas = max value
+// 9 = 1st proof removed
+// 10 = 1st commitment removed
+// 11 = max fee per blob gas = max value / blobgasperblob + 1
+// 14 = 100 blobs
+// 15 = 1000 blobs
+
+CommandLineApplication app = new() { Name = "SendBlobs" };
+
+SetupExecute(app);
+SetupDistributeCommand(app);
+SetupReclaimCommand(app);
+
+try
+{
+    app.Execute(args);
+}
+catch (CommandParsingException ex)
+{
+    Console.WriteLine(ex.Message);
+    app.ShowHelp();
+}
+
+static void SetupExecute(CommandLineApplication app)
+{
+    app.HelpOption("--help");
+
+    CommandOption rpcUrlOption = app.Option("--rpcurl <rpcUrl>", "Url of the Json RPC.", CommandOptionType.SingleValue);
+    CommandOption blobTxOption = app.Option("--bloboptions <blobOptions>", "Options in format '10x1-2', '2x5-5' etc. for the blobs.", CommandOptionType.MultipleValue);
+    CommandOption privateKeyOption = app.Option("--privatekey <privateKey>", "The key to use for sending blobs.", CommandOptionType.SingleValue);
+    CommandOption privateKeyFileOption = app.Option("--keyfile <keyFile>", "File containing private keys that each blob tx will be send from.", CommandOptionType.SingleValue);
+    CommandOption receiverOption = app.Option("--receiveraddress <receiverAddress>", "Receiver address of the blobs.", CommandOptionType.SingleValue);
+    CommandOption maxFeePerDataGasOption = app.Option("--maxfeeperdatagas <maxFeePerDataGas>", "(Optional) Set the maximum fee per blob data.", CommandOptionType.SingleValue);
+    CommandOption feeMultiplierOption = app.Option("--feemultiplier <feeMultiplier>", "(Optional) A multiplier to use for gas fees.", CommandOptionType.SingleValue);
+    CommandOption maxPriorityFeeGasOption = app.Option("--maxpriorityfee <maxPriorityFee>", "(Optional) The maximum priority fee for each transaction.", CommandOptionType.SingleValue);
```

Comment:
Consider moving to separate method

Reply:
All setup logic moved to separate class.